client/client_test.go

```diff
@@ -411,6 +413,8 @@ func (s *ClientSuite) TestUpdateRoots(c *C) {
 	}{
 		// Succeeds when there is no root update.
 		{"testdata/Published1Time", nil, map[string]int{"root": 1, "timestamp": 1, "snapshot": 1, "targets": 1}},
+		// Succeeds when client only has root.json
+		{"testdata/Published1Time_client_root_only", nil, map[string]int{"root": 1, "timestamp": 1, "snapshot": 1, "targets": 1}},
```

Comment:
Doesn't make sense. The client does have only root in the current metadata cache, but it has all files in the previous metadata cache. Should the previous cache be there?

Reply:
I removed the files in the 'previous' folder for this test . -- regardless the root update does not use the previous metadata cache, based on the way that we set-up the local repo using the "current" path.